sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -453,18 +453,18 @@ public virtual async Task<Response<CategorizedEntityCollection>> RecognizeEntiti
         /// <summary>
         /// Runs a predictive model to identify a collection of named entities
         /// in the passed-in document, and categorize those entities into types
-        /// such as person, location, or organization.  For more information on
-        /// available categories, see
-        /// <a href="https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/named-entity-types"/>.
-        /// For a list of languages supported by this operation, see
-        /// <a href="https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/language-support"/>.
-        /// For document length limits, maximum batch size, and supported text encoding, see
-        /// <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits"/>.
+        /// such as person, location, or organization.
+        /// <para>For more information on available categories, see
+        /// <a href="https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/named-entity-types"/>.</para>
```

Comment:
create issue https://github.com/Azure/azure-sdk-for-net/issues/16280 to do more cleanup